pkg/common/volume_operations.go

```diff
@@ -635,10 +635,13 @@ func (vo *VolumeOperationsImpl) deleteLVGIfVolumesNotExistOrUpdate(lvg *lvgcrd.L
 	})
 
 	drivesUUIDs := vo.k8sClient.GetSystemDriveUUIDs()
-	// if only one volume remains - remove AC first and LogicalVolumeGroup then
+	// if only one volume remains - restore AC Location, and set 0 size for sync size with drive.
+	// Then delete LogicalVolumeGroup
 	if len(lvg.Spec.VolumeRefs) == 1 && !util.ContainsString(drivesUUIDs, lvg.Spec.Locations[0]) {
-		if err := vo.k8sClient.DeleteCR(context.Background(), ac); err != nil {
-			log.Errorf("Unable to delete AC %s: %v", ac.Name, err)
+		ac.Spec.Size = 0
```

Comment:
Do we need to convert AC SC type here as well?

Reply:
Then  need to request a disk, but size 0 and there is no need for this, And I think it's better to synch AC with the disk somewhere in one place.

Reply:
We are going to delete LVG which means that AC will be converted back to non-LVG. May be it's more logical to do the conversion here and don't split the logic between two different places.

Reply:
then logic for updating AC with disk will be in several places, isn't that worse? Here we not split, we only set location, otherwise,  we will not know with which disk update in the drive reconcile. Or we can set size and sc before together before lvg remove.

Reply:
You set location to drive, but AC SC is HDDLVG. Support person will start looking at the non-existing LVG. This is confusing